pihole

```diff
@@ -456,7 +456,10 @@ Debugging Options:
                         Add '-a' to automatically upload the log to tricorder.pi-hole.net
   -f, flush           Flush the Pi-hole log
   -r, reconfigure     Reconfigure or Repair Pi-hole subsystems
-  -t, tail            View the live output of the Pi-hole log
+  -t, tail            View the live output of the Pi-hole log.
+                      Add the optional argument to filter the log
```

Comment:
"**the** optional argument" is unclear here because `-t,tail` doesn't tell me how/where. 
```suggestion
                      Add an optional argument to filter the log
```